src/core/PluginManager.cpp

```diff
@@ -165,7 +170,6 @@ namespace collabvm::core {
 				}
 			}
 
-			// FIXME: would there ever be a fatal case?
```

Comment:
Case-point: non-writable permission sets, read-only filesystems would be the most common failures for a filesystem write?

Reply:
I'd believe that a fairly large amount of people would put the binary in /var/www, where it is unable to create a plugins folder without superuser.